src/Storage.Interface/Models/DataElement.cs

```diff
@@ -95,6 +95,12 @@ public class DataElement : ChangableElement
         [JsonProperty(PropertyName = "tags")]
         public List<string> Tags { get; set; } = new List<string>();
 
+        /// <summary>
+        /// Gets or sets a list of key-value pairs that can be used to store custom metadata.
+        /// </summary>
+        [JsonProperty(PropertyName = "customMetadata")]
+        public List<KeyValueEntry> CustomMetadata { get; set; } = new();
```

Comment:
Is it relevant for all data elements or could it be nullable?

Reply:
I think it will be quite niche, and could probably be nullable. Took inspiration from the Tags prop just above it when I decided to set it to new();

Reply:
Do you think I should change it, or is it OK that it's auto initialized as an empty list?

Reply:
Storage.Interfaces is netstandard2.0 and does not support nullable reference types.

I'm still in favor of not initializing but adding a comment explaining the decision.

Reply:
Will do that.